ignite/handlers/checkpoint.py

```diff
@@ -537,18 +537,29 @@ def load_objects(to_load: Mapping, checkpoint: Mapping, **kwargs: Any) -> None:
                 checkpoint = torch.load(checkpoint_fp)
                 Checkpoint.load_objects(to_load=to_load, checkpoint=checkpoint)
 
+                # or using a string for checkpoint filepath
+
+                to_load = to_save
+                checkpoint_fp = "/tmp/models/myprefix_checkpoint_40.pth"
+                Checkpoint.load_objects(to_load=to_load, checkpoint=checkpoint_fp)
+
         Note:
             If ``to_load`` contains objects of type torch `DistributedDataParallel`_ or
             `DataParallel`_, method ``load_state_dict`` will applied to their internal wrapped model (``obj.module``).
 
         .. _DistributedDataParallel: https://pytorch.org/docs/stable/generated/
             torch.nn.parallel.DistributedDataParallel.html
         .. _DataParallel: https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html
-
         """
+
+        if isinstance(checkpoint, str):
+            checkpoint_obj = torch.load(checkpoint)
+        else:
+            checkpoint_obj = checkpoint
+
         Checkpoint._check_objects(to_load, "load_state_dict")
-        if not isinstance(checkpoint, collections.Mapping):
-            raise TypeError(f"Argument checkpoint should be a dictionary, but given {type(checkpoint)}")
+        if not isinstance(checkpoint, (collections.Mapping, str)):
+            raise TypeError(f"Argument checkpoint should be a string or a dictionary, but given {type(checkpoint)}")
```

Comment:
I assume you meant [this line](https://github.com/pytorch/ignite/blob/d9bf4e668bf65dce8b8fcc6ebd6f0769e28ce3c5/tests/ignite/handlers/test_checkpoint.py#L1073), because this is where the tests were failing. I modified the message, and all of the tests in `test_checkpoint.py` pass.